src/Agent.Plugins/TfsVCCliManager.cs

```diff
@@ -69,7 +69,12 @@ protected Task RunCommandAsync(params string[] args)
             return RunCommandAsync(FormatFlags.None, args);
         }
 
-        protected async Task RunCommandAsync(FormatFlags formatFlags, params string[] args)
+        protected Task RunCommandAsync(FormatFlags formatFlags, params string[] args)
+        {
+            return RunCommandAsync(formatFlags, false, args);
+        }
+
+        protected async Task RunCommandAsync(FormatFlags formatFlags, bool quiet, params string[] args)
```

Comment:
if you had quiet as the 3rd argument, you could make it optional and not need the wrapper version

Reply:
`args` is a `params`, so you can't put arguments after it.

Reply:
I still have so much to learn about c#

Reply:
It's the way you do variadic parameter lists in C#. I think C and others have the same restriction.